### PR TITLE
Gh 1269/select network when loading safe

### PR DESCRIPTION
**Pull request by MouazAlzahabi**

### CODE REVIEW

File: Multisig/UI/Safe Management/Add Safe/SelectNetworkViewController/SelectNetworkTableViewCell.swift [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+    @IBOutlet weak var colorImageView: UIImageView!
+    @IBOutlet weak var nameLabel: UILabel!
+    
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        nameLabel.setStyle(.primary)
+    }
+
+    override func setSelected(_ selected: Bool, animated: Bool) {
```
Comment (sche) [inline Multisig/UI/Safe Management/Add Safe/SelectNetworkViewController/SelectNetworkTableViewCell.swift:21]: please remove unused methods

File: Multisig/UI/Safe Management/Add Safe/SelectNetworkViewController/SelectNetworkViewController.swift [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        trackEvent(.networkSelect)
+    }
+
+    override func reloadData() {
+        super.reloadData()
+        currentDataTask?.cancel()
+        currentDataTask = clientGatewayService.chains(completion: { [weak self] result in
```
Comment (sche) [inline Multisig/UI/Safe Management/Add Safe/SelectNetworkViewController/SelectNetworkViewController.swift]: `chains { ... }`

File: Multisig/UI/Safe Management/Add Safe/SelectNetworkViewController/SelectNetworkTableViewCell.swift [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+//  Created by Moaaz on 6/25/21.
+//  Copyright © 2021 Gnosis Ltd. All rights reserved.
+//
+
+import UIKit
+
+class SelectNetworkTableViewCell: UITableViewCell {
+
+    @IBOutlet weak var colorImageView: UIImageView!
+    @IBOutlet weak var nameLabel: UILabel!
```
Comment (DmitryBespalov) [inline Multisig/UI/Safe Management/Add Safe/SelectNetworkViewController/SelectNetworkTableViewCell.swift:14]: per consistency, these should be private

File: Multisig/UI/UI Library/UIColor+Styles.swift [hunk 4]
```diff
@@ -30,4 +30,33 @@ extension UIColor {
     static let quaternaryBackground = UIColor(named: "quaternaryBackground")!
 }
 
+extension UIColor {
+    public convenience init?(hex: String) {
```
Comment (sche) [inline Multisig/UI/UI Library/UIColor+Styles.swift]: let's make theme colors as UIColor via Transformable

**Comment by DmitryBespalov** [comment]
Screenshot 2021-07-01 at 11 21 00

**Review by DmitryBespalov — APPROVED** [review]

**Review by DmitryBespalov — APPROVED** [review]

**Review by sche — APPROVED** [review]

**Pull request merged**